Analyzer/test/MyAnalysis.C

```diff
@@ -8,6 +8,13 @@
 #include "Analyzer/Analyzer/include/AnalyzeEventShape.h"
 #include "Analyzer/Analyzer/include/Analyze0Lep.h"
 #include "Analyzer/Analyzer/include/AnalyzeStealthTopTagger.h"
+#include "Analyzer/Analyzer/include/AnalyzeSHuHdEventSelection.h"
+#include "Analyzer/Analyzer/include/AnalyzeDataMC.h"
+#include "Analyzer/Analyzer/include/AnalyzeTest.h"
+#include "Analyzer/Analyzer/include/AnalyzeHadTrigger.h"
```

Comment:
This wont compile? Looks like you have a good amount of analyzers in your personal area. If you want them in with the master branch then add them other wise remove them and update this branch.